docroot/modules/custom/sitenow_migrate/src/Plugin/migrate/source/BaseNodeSource.php

```diff
@@ -85,6 +85,9 @@ public function prepareRow(Row $row) {
     parent::prepareRow($row);
     $moderation_state = $row->getSourceProperty('status') == 1 ? 'published' : 'draft';
     $row->setSourceProperty('moderation_state', $moderation_state);
+    if (empty($row->getSourceProperty('body_summary'))) {
```

Comment:
By default, we weren't checking/replacing summaries, so not all migrations have this. There was at least one migration earlier that specifically didn't want anything like this (I think it was a grad site?), but I think that's the minority case. Might be worth making it the default behavior going forward.

Reply:
Hmm. Confused. Wouldn't https://github.com/uiowa/uiowa/blob/028909b1bde3a4ea3a4978154d82551ab38d1135/docroot/sites/obermann.uiowa.edu/modules/obermann_migrate/src/Plugin/migrate/source/Articles.php#L57 have done this?